src/lib/eina/meson.build

```diff
@@ -199,6 +200,7 @@ sources = [
 
 if sys_windows == true
   sources += 'eina_file_win32.c'
+  sources += '_eina_fnmatch.c'
```

Comment:
Another thing here, this way `eina_fnmatch` will not be available to all builds as intended, only to a Windows one.

Reply:
Is it intended to be available to all platforms? I was thinking that others like linux will continue to use fnmatch from gnu fnmatch.h, so instead compile eina_fnmatch.c, it would `#define eina_fnmatch fnmatch`

Reply:
hmmmm, ok I think you really can do that. So your idea is to add a `#define eina_fnmatch fnmatch` after `#endif /* _WIN32 */` at `eina_fnmatch.h`?

Reply:
Oh, you have already done it. Yay